onnxruntime/core/providers/openvino/backend_utils.cc

```diff
@@ -57,7 +57,8 @@ CreateOVModel(const ONNX_NAMESPACE::ModelProto& model_proto, const GlobalContext
   const std::string model = model_proto.SerializeAsString();
   try {
     auto cnn_network = global_context.ie_core.ReadModel(model);
-    if (subgraph_context.precision == InferenceEngine::Precision::FP16) {
+    if ((subgraph_context.precision == InferenceEngine::Precision::FP16) &&
```

Comment:
Didn't HEMA add a condition here, that even if it's CPU and FP16, we need to apply ov::pass::ConvertFP32ToFP16 transformations? and that fixed issues when OVEP is run with CPU_FP16

Reply:
Done